meta-mender-raspberrypi/conf/layer.conf

```diff
@@ -11,3 +11,5 @@ BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
 BBFILE_COLLECTIONS += "mender-raspberrypi"
 BBFILE_PATTERN_mender-raspberrypi = "^${LAYERDIR}/"
 BBFILE_PRIORITY_mender-raspberrypi = "10"
+
+LAYERDEPENDS_mender-raspberrypi = "mender"
```

Comment:
"Stealth" change. Separate pull-request or at least commit. The change makes sense though.